README.md

```diff
@@ -34,6 +34,8 @@ $adapter = new CurlFtpAdapter([
   'utf8' => true,
   'ssl' => true,
   'timeout' => 90,		// connect timeout
+  'sslVerifyPeer' => 0, // using 0 is insecure, use it only if you know what you're doing
```

Comment:
why not using `false` instead of `0`?

Reply:
At least libcurl takes integer https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYPEER.html